contracts/0.8.25/vaults/VaultHub.sol

```diff
@@ -1214,6 +1204,36 @@ contract VaultHub is PausableUntilWithRoles {
         return liability > _vaultTotalValue * (TOTAL_BASIS_POINTS - _thresholdBP) / TOTAL_BASIS_POINTS;
     }
 
+    function _obligationsTooHigh(
+        VaultRecord storage _record
+    ) internal view returns (bool) {
+        return _record.redemptionShares > 0 || _unsettledLidoFeesValue(_record) >= MIN_BEACON_DEPOSIT;
+    }
+
+    function _minPartialWithdrawalAmount(
```

Comment:
```suggestion
    function _uncoveredObligations(
```

Let's extend obligations to:
- maintain healthiness
- fulfill redemptions
- pay fees

But need to check other places where we use obligations term to check if they are aligned